pom.xml

```diff
@@ -399,6 +404,37 @@ under the License.
 				<spotless.skip>true</spotless.skip>
 			</properties>
 		</profile>
+
+		<profile>
+			<id>mongodb4</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<properties>
+				<mongodb.version>4.0.10</mongodb.version>
+			</properties>
+		</profile>
+
+		<profile>
+			<id>mongodb5</id>
+			<properties>
+				<mongodb.version>5.0.2</mongodb.version>
+			</properties>
+		</profile>
+
+		<profile>
+			<id>mongodb6</id>
+			<properties>
+				<mongodb.version>6.0.6</mongodb.version>
```

Comment:
Will it be better if we test each major MongoDB release with the latest patch version?

Reply:
Good idea.